src/components/AssetsComponent.jsx

```diff
@@ -93,23 +94,26 @@ export default class AssetsComponent extends Component {
 
   render() {
     const { assets } = this.state;
-    const { status } = this.props;
+    const { status, exportAsset, exportAssetsAction } = this.props;
     const totalPages = this.handlePageTotal();
     const currentAssets = `page_${this.props.activePage}`;
 
     return (
       <Fragment>
-        {
-          <Filter
-            activePage={this.props.activePage}
-            limit={this.state.limit}
-            filterData={this.props.filterData}
-            selected={this.props.selected}
-            filterSelection={this.props.filterSelection}
-            filterAction={this.props.getAssetsAction}
-            disabled={this.props.isLoading}
-          />
-        }
+        <ExportAsset
+          assets={this.props.assetsList[currentAssets] || assets}
+          exportAsset={exportAsset}
+          exportAssetsAction={exportAssetsAction}
+        />
+        <Filter
+          activePage={this.props.activePage}
+          limit={this.state.limit}
+          filterData={this.props.filterData}
+          selected={this.props.selected}
+          filterSelection={this.props.filterSelection}
+          filterAction={this.props.getAssetsAction}
+          disabled={this.props.isLoading}
+        />

```

Comment:
Please destructure all state and props objects.

Reply:
Removed, thanks